Domain.Core/Models/UserInfo.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Domain.Core.Models
+{
+    public class UserInfo
+    {
+        public Guid UserId { get; set; }
+        public string PictureSrc { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+
+        public User User { get; set; }
+        public Farm Farm { get; set; }
+        public IEnumerable<FarmFriend> FarmFriends { get; set; }
```

Comment:
Why we work with IEnumerable instead of ICollection or IList, or List?

Reply:
To be honestly, i haven't ever thought about it. In every example or tutorial, that i saw, i met IEnumerable and i haven't ever had any troubles with it.